perfkitbenchmarker/benchmark_spec.py

```diff
@@ -139,59 +195,35 @@ def ConstructVirtualMachines(self):
     """Constructs the BenchmarkSpec's VirtualMachine objects."""
     vm_group_specs = self.config.vm_groups
 
-    zone_index = 0
+    os_type = vm_group_specs[vm_group_specs.keys()[0]].os_type
+    if os_type == os_types.JUJU:
```

Comment:
Ok, I'll take a look at that. My assumption was that, when the os_type == JUJU, all of the vm_groups would share that. Is there a scenario when that is not the case?